### PR TITLE
[SYCL][Bindless][E2E] fix double free in vulkan interop unsampled test

### PR DESCRIPTION
**Pull request by cppchedy**
* fix double free in vulkan interop unsampled test

### CODE REVIEW

**Comment by cppchedy** [comment]
[UR PR](https://github.com/oneapi-src/unified-runtime/pull/2299)

File: sycl/cmake/modules/FetchUnifiedRuntime.cmake [hunk 1]
```diff
@@ -117,7 +117,7 @@ if(SYCL_UR_USE_FETCH_CONTENT)
   endfunction()
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/UnifiedRuntimeTag.cmake)
+  set(UNIFIED_RUNTIME_TAG 698cd9974c18874bf6a7e9ec78507c30b9deeca7)
```
Comment (DBDuncan) [inline sycl/cmake/modules/FetchUnifiedRuntime.cmake:120]: I believe now you need to instead edit `UnifiedRuntimeTag.cmake` with the new commit and leave this line the same.

Comment (cppchedy) [reply sycl/cmake/modules/FetchUnifiedRuntime.cmake]: ah. thank you. I missed this change.

**Review by DBDuncan — APPROVED** [review]

**Comment by ProGTX** [comment]
Friendly ping @intel/unified-runtime-reviewers . Tests passed and the UR PR is ready to merge.

**Comment by kbenzie** [comment]
> Friendly ping @intel/unified-runtime-reviewers . Tests passed and the UR PR is ready to merge.

We'll try and fit this in as soon we can once we've merged all the features for the next release.

**Review by martygrant — APPROVED** [review]

**Pull request merged**